flows/instruments/instruments.py

```diff
@@ -432,7 +432,13 @@ def get_obstime(self):
         return obstime
 
     def get_photfilter(self):
-        return self.image.header['CCDFLTID']
+        instrument = self.image.header['INSTRUME']
+        if instrument == 'ANDICAM-IR':
```

Comment:
instrument is already set to be ANDICAM-CCD, so create a new AndiCamIR instrument for this. If the instrument field is AndiCamIR this class won't even detect that the image belongs to the AndiCam pipeline.

Reply:
I tried something like:
class AndiCamIR(Instrument):
    andicam = AndiCam()
    siteid = andicam.siteid  
    instrument = 'ANDICAM-CCD’
    unique_headers = andicam.unique_headers

    def get_obstime(self):
        return andicam.get_obstime()

    def get_photfilter(self):
        return self.image.header['IRFLTID']
                       
However, this does not seem to work. In the current commit, I added some spaces to a file, but it seems it does not pass the checks. Is something off with the pipeline?